src-java/network-topology/network-storm-topology/src/main/java/org/openkilda/wfm/topology/network/storm/bolt/sw/SwitchHandler.java

```diff
@@ -211,6 +218,11 @@ private Values makeRerouteTuple(SwitchId switchId) {
                 getCommandContext().fork("reroute"));
     }
 
+    private Values makeUpdateSingleSwitchFlowsTuple(SwitchId switchId, SwitchStatus status) {
```

Comment:
Incorrect pattern usage. Each `make*Tuple` method serve corresponding stream. These methods make abstraction over stream fields set, that's why they are bound to streams. So if you push message into `reroute` stream, you must use `makeRerouteTuple(...)`.